meson.build

```diff
@@ -909,7 +909,8 @@ if want_bpf_framework == 'false'
 else
         clang = find_program('clang', required : bpf_framework_required)
         llvm_strip = find_program('llvm-strip', required : bpf_framework_required)
-        bpftool = find_program('bpftool', required : bpf_framework_required)
+        # Debian installs this in /usr/sbin/ which is not in $PATH
+        bpftool = find_program('bpftool', '/usr/sbin/bpftool', required : bpf_framework_required)
```

Comment:
We could but it's a bit of a pain. Arguably bpftool should not be in sbin, as you don't have to be root to do a whole bunch of stuff, but can't change that.
Meson 0.53 takes a `dirs` param in `find_program`, maybe we can go for this now and switch to that after we bump the minimum version?
https://mesonbuild.com/Reference-manual.html#find_program

Reply:
Let's do that.

Reply:
maybe add  FIXME comment here suggesting this?

i'd consider this a debian packaging bug though... if we can make use of the tool unprivileged in a sensible way it shouldn#t be in sbin

Reply:
Added - yeah, I imagine what happened is that initially it required privileges, and then unprivileged stuff was added. Moving files around in packages is not trivial due to backward compat and upgrade paths. I'll see what it can be done for the Debian 12 timeframe.

Reply:
sbin is a mistake anyway, exactly for reasons like this. in an ideal world sbin would just become a symlink to bin. another political endeavour one should fight through one day

Reply:
opened a ticket: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=989083

Reply:
fair point - what does fedora do?

Reply:
Looks like it does the same thing as Debian:

```
$ dnf provides /usr/*bin/bpftool
bpftool-5.8.11-300.fc33.x86_64 : Inspection and simple manipulation of eBPF programs and maps
Repo        : fedora
Matched from:
Filename    : /usr/sbin/bpftool

bpftool-5.11.16-200.fc33.x86_64 : Inspection and simple manipulation of eBPF programs and maps
Repo        : updates
Matched from:
Filename    : /usr/sbin/bpftool
```

Reply:
And the same thing goes for Arch Linux as well:

```
# pacman -Ql bpf | grep bin/bpftool
bpf /usr/bin/bpftool
```

so, yeah, if this should be changed, it should go through the upstream first.

Reply:
And for completion, Arch Linux manually "fixes" the `bpftool` path:

https://github.com/archlinux/svntogit-community/blob/e877964e81ad5fa292048e9523984096e920be9a/trunk/PKGBUILD#L287-L289

Reply:
I see - I guess the default PATH is the difference then